lib/rucio/daemons/bb8/common.py

```diff
@@ -101,12 +101,98 @@ def rebalance_rule(parent_rule_id, activity, rse_expression, priority, source_re
                           comment=parent_rule['comments'] if not comment else comment,
                           ask_approval=False,
                           asynchronous=False,
+                          ignore_account_limit=True,
                           priority=priority)[0]
 
-    update_rule(rule_id=parent_rule_id, options={'child_rule_id': child_rule, 'lifetime': 0})
+    update_rule(rule_id=parent_rule['id'], options={'child_rule_id': child_rule, 'lifetime': 0})
     return child_rule
 
 
+def __dump_url(rse):
+    """
+    getting rse dump with different methods according to settings
+
+    :param rse          RSE where the dump is released.
+    """
+
+    # get the date of the most recent dump
+    dump_production_day = config_get('bb8', 'dump_production_day', raise_exception=False, default='Sunday')
```

Comment:
Do you expect that the config is always updated for this? I wonder if this is necessary, or if it is enough to just iterate back from today, one day at a time?